src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  deleteTodo: (id: number) => void,
```

Comment:
Consider renaming 'deleteTodo' to 'onDeleteTodo' to follow the convention of using 'on' prefix for event handlers